config.yml

```diff
@@ -37,9 +37,6 @@ genesis:
     gov:
       deposit_params:
         min_deposit: [{ amount: "1", denom: "usge" }]
-
-      voting_params:
```

Comment:
@scorpioborn I was talking about adding a comment on what should the voting_period means and what should be its value